### ISSUE TITLE
Needs a .gitignore file

### ISSUE THREAD

**Issue by Shinobu1337** [kelseyhightower/nocode#2484]
.gitignore files are an extremely important part of Git. We need to add one here.

**Comment by dxvargas** [comment]
I completely agree. It should ignore everything, like this every piece of code will be ignored.
I've created a .gitignore and added a "*" inside. but now I can't commit it because it's being ignored.

**Comment by Shinobu1337** [comment]
lol well, apparently the mobile version of GitHub lets you click the close issue button even before the page loads
